### PR TITLE
Seed ph module from Cmodule template

### PR DESCRIPTION
**Pull request by maurin**
Hey guys, this is a new module for iterfacing with ph4567 pH sensor.

The sensor communicates of i2c.  I used the Arduino I2C libraries.  Please check the datasheet to ensure that all of the initialization registers are correctly set for our range.

### CODE REVIEW

File: rover/Arduino/sketches/Poseidon/Cph.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class CModule
+{
+public:
+	// Methods
+	CModule();
+
+	// Pure virtuals - These methods must be instantiated in your derived class!
+	virtual void Initialize() = 0;
+	virtual void Update( CCommand& commandIn ) = 0;
+};
```
Comment (joshnarayan) [inline rover/Arduino/sketches/Poseidon/Cph.h:15]: very nice curly brace

Comment (piersohannah21) [reply rover/Arduino/sketches/Poseidon/Cph.h:15]: comment test

File: rover/Arduino/sketches/Poseidon/Cph.cpp [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Includes
+#include <Arduino.h>
+
+#include "CModule.h"
+#include "NModuleManager.h"
+
+CModule::CModule()
```
Comment (CaitlynFong) [inline rover/Arduino/sketches/Poseidon/Cph.cpp:7]: class name supposed to be PH

File: rover/Arduino/sketches/Poseidon/Cph.cpp [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+{
```
Comment (GarrettWeber) [inline rover/Arduino/sketches/Poseidon/Cph.cpp:8]: where are all the comments?

File: rover/Arduino/sketches/Poseidon/Cph.h [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "CCommand.h"
+#include <Arduino.h>
+
+class CModule
+{
+public:
+	// Methods
+	CModule();
```
Comment (GarrettWeber) [inline rover/Arduino/sketches/Poseidon/Cph.h:10]: awesome function names

File: rover/Arduino/sketches/Poseidon/Cph.cpp (hunk 2 above)
Comment (TheMage55) [reply rover/Arduino/sketches/Poseidon/Cph.cpp:7]: comment test?

File: rover/Arduino/sketches/Poseidon/Cph.h (hunk 1 above)
Comment (maurin) [reply rover/Arduino/sketches/Poseidon/Cph.h:15]: Oh yeah.  My favorite curly style.

**Review by GarrettWeber — COMMENTED** [review]
this code does nothing, great job!

**Review by CaitlynFong — APPROVED** [review]
All righty, looks good! Ready to merge.

**Pull request merged**